### PR TITLE
style: fix blog formatting of quotes + instruction items

### PR DESCRIPTION
**Pull request by dennyscode**
Ticket: https://lifi.atlassian.net/browse/LF-9653

### CODE REVIEW

File: src/components/Blog/CustomRichBlocks.tsx [hunk 1]
```diff
@@ -214,7 +225,7 @@ export const CustomRichBlocks = ({
                         strikethrough={el.props.strikethrough}
                         underline={el.props.underline}
                         bold={el.props.bold}
-                        key={`blog-paragraph-line-${index}-${lineIndex}`}
+                        key={generateKey('blog-paragraph-line')}
```
Comment (oktapodia) [inline src/components/Blog/CustomRichBlocks.tsx:228]: This key might conflicts with another one as it does not have any variable in anymore

**Review by oktapodia — CHANGES_REQUESTED** [review]
Just a quick comment

File: src/components/Blog/CustomRichBlocks.tsx (hunk 1 above)
Comment (dennyscode) [reply src/components/Blog/CustomRichBlocks.tsx]: Thanks Sebastien! I have undone those changes - but out of curiosity: Does the generateKey not create a random key anyways, already preventing a clash with any existing key?

**Comment by dennyscode** [comment]
**Handle "Quote" rich-text elements:**

Previously:
image

Fixed here:
image

**Comment by dennyscode** [comment]
**Handle Instruction-Item colors on dark-mode:**

Previously:
image

Fixed here:
image

File: src/components/Blog/CustomRichBlocks.tsx [hunk 2]
```diff
@@ -102,6 +102,17 @@ export const CustomRichBlocks = ({
           );
       }
     },
+    quote: ({ children }: any) => {
```
Comment (oktapodia) [inline src/components/Blog/CustomRichBlocks.tsx:105]: Would be good to type it

File: src/components/Blog/CustomRichBlocks.tsx (hunk 1 above)
Comment (oktapodia) [reply src/components/Blog/CustomRichBlocks.tsx]: Hum, you are actually right

**Review by admsk2 — APPROVED** [review]

File: src/components/Blog/CustomRichBlocks.tsx (hunk 2 above)
Comment (dennyscode) [reply src/components/Blog/CustomRichBlocks.tsx]: Thanks! I added some more types now for the quotes + paragraph elements.

**Review by oktapodia — APPROVED** [review]
LGTM

**Pull request merged**